.github/workflows/full-checks.yml

```diff
@@ -0,0 +1,202 @@
+name: Full Checks
+
+on:
+  schedule:
+    - cron: '0 1 * * *' # run at 1AM UTC
+  workflow_dispatch:
+
+env:
+  HOMEBREW_NO_INSTALL_CLEANUP=1: 1 # Disable cleanup for homebrew, we don't need it on CI
+
+jobs:
+  build-and-test-release:
+    name: Test LLC + UI (Release)
+    runs-on: macos-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set build image var
+      run: echo "ImageVersion=$ImageVersion"  >> $GITHUB_ENV
+    - name: Cache RubyGems
```

Comment:
Why don't we add an action for this too?

Reply:
The build-image-var action we can create it yes. 👍

Reply:
✅